### PR TITLE
Update Software airgap instructions for runtime

### PR DESCRIPTION
**Pull request by rob-1126**
Renames the sample web server deployment that services astronomer-certified to astronomer-releases and adds instructions for creating the secret to hold the Runtime file and configuration section (basically mirrors whats there for -certified).

### CODE REVIEW

File: software/install-airgapped.md [hunk 1]
```diff
@@ -214,33 +223,33 @@ To complete this setup:
...
+    The updates JSON will be accessible by the service name from pods in the Kubernetes cluster via `http://astronomer-releases.astronomer.svc.cluster.local/astronomer-certified.json`.
 
 To validate if the updates JSON is accessible you have several options:
 
 - If an image with `curl` is available in your network, you can run:
 
     ```bash
     $ kubectl run --rm -it [container name] --image=[image] --restart=Never -- /bin/sh
-    $ curl http://astronomer-certified.astronomer.svc.cluster.local/astronomer-certified.json
+    $ curl http://astronomer-releases.astronomer.svc.cluster.local/astronomer-certified.json
```
Comment (pritt20) [inline software/install-airgapped.md:237]: Hey @rob-1126 ,

Thanks for modifying this doc.

Under validation steps at line 231, I see that all validation are for astronomer-certified.json, Should we also add similar validations for astronomer-release as well?

line 237 : curl http://astronomer-releases.astronomer.svc.cluster.local/astronomer-runtime.json
line 245:  curl http://localhost:8001/api/v1/namespaces/astronomer/services/astronomer-releases/astronomer-runtime.json
line 252: curl http://astronomer-releases.astronomer.svc.cluster.local/astronomer-runtime.json

**Comment by rob-1126** [comment]
Thanks, looks like will need an addition to cover over-riding the following as well under the helm subsection. Will poke a bit more tomorrow:

```
    # Default runtime repository
    defaultRuntimeRepository: quay.io/astronomer/astro-runtime

    # Default runtime images
    runtimeImages:
      airflow:
        repository: quay.io/astronomer/astro-runtime
      flower:
        repository: quay.io/astronomer/astro-runtime
```

File: software/install-airgapped.md (hunk 1 above)
Comment (rob-1126) [reply software/install-airgapped.md]: Thanks, just pushed. The validation examples we had looked to be using the wrong filename so I fixed that as well.

**Review by jwitz — APPROVED** [review]
Ah sorry @rob-1126 missed this last week, but this looks great!

**Pull request merged**